libs/SalesforceSDKCommon/SalesforceSDKCommon/Classes/Logger/SFLogger.h

```diff
@@ -365,6 +367,12 @@ NS_SWIFT_NAME(SalesforceLogger)
  */
 + (void)setInstanceClass:(Class<SFLogging>)logger;
 
+/**
+ * Sets the log receiver factory which creates log receivers.
+ * @param logReceiverFactory The log receiver factory
+ */
++ (void)setLogReceiverFactory:(id<SFLogReceiverFactory>)logReceiverFactory;
```

Comment:
On Android we also created a static method for the registration of the new factory instance.